app/models/lookup_keys/lookup_key.rb

```diff
@@ -144,6 +144,12 @@ def overridden?(host)
     host.lookup_values.any? { |lv| lv.lookup_key_id == id }
   end
 
+  def overridden_value(host)
```

Comment:
If you are going to add this change, 3 lines above you can have the previous method work in terms of this one:

```ruby
overridden_value(host).present?
```